.gitlab/includes/clang11_pipeline.yml

```diff
@@ -20,26 +20,45 @@ include:
                   -DPIKA_WITH_MAX_CPU_COUNT=256 -DPIKA_WITH_MALLOC=system \
                   -DPIKA_WITH_SPINLOCK_DEADLOCK_DETECTION=ON"
 
-clang11_spack_image:
+clang11_spack_compiler_image:
   stage: spack_configs
   needs: [base_spack_image]
   extends:
     - .container-builder
     - .variables_clang11_config
   before_script:
     - export DOCKERFILE_SHA=`sha256sum $DOCKERFILE | head -c 16`
-    - CONFIG_TAG=`echo $DOCKERFILE_SHA-$ARCH-$BASE_IMAGE-$COMPILER-$SPACK_SPEC | sha256sum - | head -c 16`
+    - CONFIG_TAG=`echo $DOCKERFILE_SHA-$BASE_IMAGE-$SPACK_SPEC | sha256sum - | head -c 16`
```

Comment:
I agree with them being included as you describe, but that inclusion is not obvious right here without knowing what's in `SPACK_SPEC`. My preference would be to include them explicitly anyway since there's no downside to it. The upside is that it makes it straightforward to check if all the `DOCKER_BUILD_ARGS` are included in the tag.

Reply:
It looks like you've changed back in other configurations, but not in this one. That's probably not intentional?